bloc-did-method.md

```diff
@@ -8,99 +8,165 @@ The `did:bloc` DID method allows groups of independent organizations to share cu
  - update to the latest version that has sufficient endorsement?
+ 
+- Policy may specify that a number of independent queries have to be made (queries made to endpoints 
+  from different organizations in the consortium). TODO is to justify why.
+  - This isn't about independent queries for regular use, but independent queries when bootstrapping trust, to verify that stakeholders named in the consortium config have signed the consortium config (and signed their own stakeholder config) with keys that are associated with did docs on the ledger
+  - maybe we can suggest resolving stakeholder X's did doc using stakeholder Y's endpoint, to verify the stakeholders are on the same ledger?
+    - but then, if they all sign the same consortium config, which points to all of them, we don't need this complication
 --> 
+### Bootstrapping Trust
+
+The Bloc DID method is designed to offer several mechanisms for a resolver to confirm that it can trust the stakeholders to provide the DID doc ledger service. 
 
-### CRUD Operation Definitions
+In each mechanism, the resolver has the option to cache verified configuration files for a period of time (the cache lifetime being specified in the configuration), and in some of these, the resolver can automatically fetch and verify updates.
+
+#### Automatic Bootstrapping
+Automatic bootstrapping relies on the assumption of trust in the consistency of the owner of a web domain. In resolving a did:bloc DID, the resolver assumes the consortium domain in the DID is safe, and that the stakeholder domains within the consortium config are also safe and correct.
+
+On a first connection by a resolver to the ledger, the bloc DID method provides a mechanism for the resolver to establish trust in the consortium and stakeholders.
+
+A resolver can begin by assuming trust in the consortium domain contained within a bloc DID. The config found there will inform the resolver about all the stakeholder organizations within the consortium. The resolver can trust the authenticity of these stakeholders based on their domain name registration, as well as their inclusion within the consortium config. 
+
+The resolver must verify at least M stakeholders, to retrieve their endpoints and confirm their signatures on the consortium config.
+
+For each stakeholder, the resolver fetches the stakeholder config from their domain. This includes a list of Sidetree endpoints managed by the stakeholder. With one of these, the resolver can resolve the stakeholder's DID to retrieve the stakeholder's DID doc. This DID Doc contains the key used to verify the signature on the stakeholder's config file, and one of the signatures on the consortium config file.
+
+Once the resolver has verified M signatures on the consortium, via this process, it can cache the configuration info, having established trust. The request to resolve a DID can then proceed, and further requests can proceed directly to a saved endpoint.
```

Comment:
```suggestion
Once the resolver has verified M signatures on the consortium, via this process, it can cache the configuration info, having established trust. The request to resolve a DID can then proceed, and further requests can proceed directly to a cached endpoint.
```